src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double secondLeg, double firstLeg) {
+        super(color);
+        this.secondLeg = secondLeg;
+        this.firstLeg = firstLeg;
+    }
+
+    public double getFirstLeg() {
+        return firstLeg;
+    }
+
+    public void setFirstLeg(double firstLeg) {
+        this.firstLeg = firstLeg;
+    }
+
+    public double getSecondLeg() {
+        return secondLeg;
+    }
+
+    public void setSecondLeg(double secondLeg) {
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void moreInfo() {
+        System.out.println("Figure: RightTriangle");
+        System.out.println("FirstLeg: " + firstLeg + " units");
+        System.out.println("SecondLeg: " + secondLeg + " units");
+    }

```

Comment:
Use only English in messages/code. Please ensure that all messages are consistent and professional. For example, instead of 'units', you might use 'Length of the first leg: '.